scanner/indexer/indexer_test.go

```diff
@@ -0,0 +1,62 @@
+package indexer
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseImageURL(t *testing.T) {
```

Comment:
nit: `TestParseContainerImageURL` just to match the function being tested

Reply:
8bba0ff8